packages/desktop-client/src/components/budget/util.ts

```diff
@@ -37,15 +37,27 @@ export function makeAmountGrey(value: number | string) {
     : null;
 }
 
-export function makeAmountStyle(value: number) {
-  const greyed = makeAmountGrey(value);
-  if (greyed) {
-    return greyed;
-  }
-
+export function makeAmountStyle(
+  value: number,
+  goalValue: number,
+  budgetedValue: number,
```

Comment:
Make the goal and budget values optional:
```suggestion
  goalValue?: number,
  budgetedValue?: number,
```

Reply:
added